coin/coin_presentation/src/main/java/com/pavluyk/coin_presentation/activity/CoinMonitoringFragment.kt

```diff
@@ -31,16 +29,16 @@ class CoinMonitoringFragment : Fragment() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        viewModel.coinDataLiveData.observe(this, Observer {
+        viewModel.coinDataLiveData.observe(this) {
             it?.let { adapter.setData(it) }
```

Comment:
does this really can be null? doublecheck plz
if yes, try avoid 'it' notation, this drops readability dramatically, better do "coins ->"